packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -1141,7 +1147,24 @@ contract VotingV2 is
                 voteInstance.resultComputation.getResolvedPrice(_computeGat(priceRequest.lastVotingRound));
             return (true, resolvedPrice, "");
         } else if (requestStatus == RequestStatus.Future) return (false, 0, "Price is still to be voted on");
-        else return (false, 0, "Price was never requested");
+        else {
```

Comment:
nit: this is purely an aesthetic expendable comment, but I wanted to know what you thought about it. I find the "early return" pattern more readable and understandable.  So `_getPriceOrError` would look like this:

```
function _getPriceOrError(
        bytes32 identifier,
        uint256 time,
        bytes memory ancillaryData
    )
        internal
        view
        returns (
            bool,
            int256,
            string memory
        )
    {
        PriceRequest storage priceRequest = _getPriceRequest(identifier, time, ancillaryData);
        uint256 currentRoundId = voteTiming.computeCurrentRoundId(getCurrentTime());

        RequestStatus requestStatus = _getRequestStatus(priceRequest, currentRoundId);
        if (requestStatus == RequestStatus.Active) return (false, 0, "Current voting round not ended");

        if (requestStatus == RequestStatus.Resolved) {
            VoteInstance storage voteInstance = priceRequest.voteInstances[priceRequest.lastVotingRound];
            (, int256 resolvedPrice) =
                voteInstance.resultComputation.getResolvedPrice(_computeGat(priceRequest.lastVotingRound));
            return (true, resolvedPrice, "");
        }

        if (requestStatus == RequestStatus.Future) return (false, 0, "Price is still to be voted on");

        (bool previouslyResolved, int256 previousPrice) =
            _getPriceFromPreviousVotingContract(identifier, time, ancillaryData);

        if (previouslyResolved) return (true, previousPrice, "Returned from previous contract");

        return (false, 0, "Price was never requested");
    }
```

Reply:
I actually far prefer your layout. i was trying to keep it the way it was before but what you've got is actually much cleaner.